cmd/ceremony/cert_test.go

```diff
@@ -368,6 +398,63 @@ func TestVerifyProfile(t *testing.T) {
 			},
 			certType: ocspCert,
 		},
+		{
+			profile: certProfile{
+				NotBefore:          "a",
+				NotAfter:           "b",
+				SignatureAlgorithm: "c",
+				CommonName:         "d",
+				Organization:       "e",
+				Country:            "f",
+				IssuerURL:          "g",
+				OCSPURL:            "h",
```

Comment:
I'd only include one incorrect field in each of these tests. As it is, you're testing implementation (what order are these fields checked in?) instead of just correctness (each of these three fields should cause errors independently).